<launch>

  <node
      name="ros_traj_consum_interpol_interface_Yin"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_yin.yaml"
        type="str"/>

    <param
      name="robot_name"
      value="yin_visual"
      type="str"/>

  </node>


  <arg name="dress" default="false"/>

  <group if="$(arg dress)">
    <!-- Testing node trajectory publisher -->
    <node name="test_human_dressing_motion"
        pkg="ros_pybullet_interface"
        type="test_human_dressing_motion.py"
        output="screen">

       <param
          name="robot_name"
          value="yin_visual"
          type="str"/>

    </node>

  </group>

  <group unless="$(arg dress)">
    <!-- Testing node trajectory publisher -->
    <node name="test_move_robot_to_initial_for_dress"
        pkg="ros_pybullet_interface"
        type="test_move_robot_to_initial_dressing_motion.py"
        output="screen">

       <param
          name="robot_name"
          value="yin_visual"
          type="str"/>

    </node>

  </group>



</launch>
